tools.mk

```diff
@@ -52,4 +52,7 @@ $(GOX): ## Build gox
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL)  github.com/mitchellh/gox@v1.0.1
 
 $(GCOV2LCOV): ## Build gcov2lcov
-	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/jandelgado/gcov2lcov@v1.0.2
\ No newline at end of file
+	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) github.com/jandelgado/gcov2lcov@v1.0.2
+
+.PHONY: install-tools
+install-tools: $(KIND) $(KUSTOMIZE) $(CONTROLLER_GEN) $(CONVERSION_GEN) $(GOLANGCI_LINT) $(GOLINT) $(GOX) $(GCOV2LCOV)
```

Comment:
What calls this?

Reply:
humans!

Reply:
This target used to exist a while ago but disappeared.

Reply:
Ah, probably accidentally removed in some refactor. Sounds good.